addon/core/handler.ts

```diff
@@ -81,23 +91,123 @@ export default class TableHandler {
     throw new Error('NotImplemented');
   }
 
-  // @ts-ignore
-  applyFilters(column: Column, filters: Filter[]): Promise<any> {
-    throw new Error('NotImplemented');
+  /**
+   * Set filters on a given column.
+   *
+   * @param {Column} column - The column on which to apply filters
+   * @param {Filter[]} filters - The array of filters to apply to the column
+   * @returns {Promise<any>}
+   */
+  async applyFilters(column: Column, filters: Filter[]): Promise<any> {
+    const _filters = filters.reduce((acc, v) => {
+      const filterWithSameKey = acc.find((filter) => filter.key === v.key)
+
+      if (filterWithSameKey) {
+        filterWithSameKey.value = v.value;
+      } else {
+        acc.push(v);
+      }
+
+      return acc;
+    }, [...column.filters])
+
+    column.filters = _filters;
+
+    return this.tableManager.upsertColumns({ columns: this.columns }).then(({ columns }) => {
+      this.columns = columns;
+      this.currentPage = 1;
+      this.fetchRows();
```

Comment:
this seems be duplicated, maybe have an `resetColumns(columns)` method or something like that ^^